src/test/isMobilePhoneNumber.spec.ts

```diff
@@ -1,9 +1,9 @@
 import { expect } from 'chai'
-import { IsPhoneNumber } from '../main/isPhoneNumber'
+import { IsMobilePhoneNumber } from '../main/isMobilePhoneNumber'
 import { validateSync } from 'class-validator'
 
 class PhoneNumberTest {
```

Comment:
To keep it consistent we could rename test domain class to be `MobilePhoneNumberTest `